packages/datatrak-web-server/src/app/createApp.ts

```diff
@@ -54,11 +54,12 @@ export function createApp() {
     attachModels: true,
   })
     .useSessionModel(DataTrakSessionModel)
+
     .useAttachSession(attachSessionIfAvailable)
     .attachApiClientToContext(authHandlerProvider)
     .post<SubmitSurveyRequest>('submitSurvey', handleWith(SubmitSurveyRoute))
     .get<UserRequest>('getUser', handleWith(UserRoute))
-    .get<SingleEntityRequest>('entity/:projectCode/:entityCode', handleWith(SingleEntityRoute))
```

Comment:
I'm not sure about this one - the projectCode is useful for determining things like the parentCode etc. I know that if the survey is being accessed publicly then the user doesn't have a project assigned, but could we get it from the survey itself?